device/globals/game.gd

```diff
@@ -251,6 +251,18 @@ func _process(time):
 
 	player.walk_to(player.get_position() + dir * 20)
 
+func _input(ev):
+	if ProjectSettings.get_setting("escoria/ui/tooltip_follows_mouse"):
+		# Must verify `position` is there, key inputs do not have it
+		if vm.hover_object and "position" in ev:
+			var tooltip = get_node("hud_layer/hud/tooltip")
+			var pos = ev.position
+			var pos_x = ev.position.x - (tooltip.get_size().x / 2)
+			var pos_y = ev.position.y - (tooltip.get_size().y)
+			pos.x = pos_x
+			pos.y = pos_y
+			tooltip.set_position(pos)
```

Comment:
My vector math is a bit rusty, but I think this can be written as `var pos = ev.position - tooltip.get_size() / Vector2(2, 1)`

Reply:
Something new every day! That works and looks like the preferred way. Force-push incoming soon.